### PR TITLE
[DOCFIX] Add documentation for Tachyon fault tolerant in standalone mode

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

**Comment by gpang** [comment]
Jenkins, test this please

File: docs/Running-Tachyon-Fault-Tolerant.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+layout: global
+title: Tachyon Standalone with Fault Tolerance
+nickname: Tachyon Standalone with Fault Tolerance
+group: User Guide
+priority: 3
+---
+
+Fault Tolerance in Tachyon is based upon a multi-master approach where multiple master processes
+are run. One of these processes is elected the leader and is used by all workers and clients as the
+primary point of contact. The other masters act as standbys using the shared journal to ensure that
```
Comment (uronce-cc) [inline docs/Running-Tachyon-Fault-Tolerant.md]: Would be better to specify that `the shared journal` is generated by leader master, other masters just read the journal to keep the same state with master, also, it's good to know where the journal should be put.

Comment (uronce-cc) [inline docs/Running-Tachyon-Fault-Tolerant.md]: nvm, I found the `journal` is better explained below.

File: docs/Running-Tachyon-Fault-Tolerant.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+In the following sections we refer to an "externally visible address". This is simply the address of
+an interface on the machine being configured that can be seen by other nodes in the Tachyon cluster.
+On EC2, you should the `ip-x-x-x-x` address. In particular, don't use `localhost` or `127.0.0.1`, as
+other nodes will then be unable to reach your node.
+
+### Configuring Fault Tolerant Tachyon
+
+In order to enable fault tolerance for Tachyon, additional configuration settings must be set for
+Tachyon masters, workers, and clients. In `conf/tachyon-env.sh`, these java options must be set:
```
Comment (uronce-cc) [inline docs/Running-Tachyon-Fault-Tolerant.md]: These configurations can also be put in `conf/tachyon-site.properties`.

File: docs/Running-Tachyon-Fault-Tolerant.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+### Client Configuration
+
+No additional configuration parameters are required for fault tolerant mode. As long as both:
+
+    -Dtachyon.zookeeper.enabled=true
+    -Dtachyon.zookeeper.address=[zookeeper_hostname]:2181
+
+are set appropriately for your client application, the application will be able to consult with
+ZooKeeper for the current leader master.
```
Comment (uronce-cc) [inline docs/Running-Tachyon-Fault-Tolerant.md]: it's a bit confusing. In my past experience in using spark, with tachyon-ft, `sc.textFile("tachyon://{one of the master addresses}:19998/file")` will get the data, when the above two parameters are not set in spark job. In the code path, if the master address used is not leader master, then after trying, the request will be transferred to the leader master, I remember it works like this before.

Comment (gpang) [inline docs/Running-Tachyon-Fault-Tolerant.md]: hrmmm, interesting. How does the client discover the real master?

Comment (uronce-cc) [inline docs/Running-Tachyon-Fault-Tolerant.md]: Yes, I'm wrong, in current code, if the master is not leader, the thrift service won't be started. Maybe I contained these configurations in client. Then it's good to specify what Tachyon URI can be used in client side when in tachyon-ft mode, in current code, the hostname and port in the Tachyon URI can be anything due to https://github.com/amplab/tachyon/blob/master/common/src/main/java/tachyon/MasterClientBase.java#L59

File: docs/Running-Tachyon-Fault-Tolerant.md (hunk 2 above)
Comment (gpang) [inline docs/Running-Tachyon-Fault-Tolerant.md]: Added info about this.

File: docs/Running-Tachyon-Fault-Tolerant.md (hunk 3 above)
Comment (gpang) [inline docs/Running-Tachyon-Fault-Tolerant.md]: So, how should I update the docs for configuring the client?

**Comment by gpang** [comment]
I updated PR. Thanks!

**Comment by gpang** [comment]
Jenkins, test this please

File: docs/Running-Tachyon-Fault-Tolerant.md (hunk 3 above)
Comment (gpang) [inline docs/Running-Tachyon-Fault-Tolerant.md]: @uronce-cc , does anything have to be added for the client configuration section?

**Comment by calvinjia** [comment]
LGTM

**Comment by uronce-cc** [comment]
LGTM

**Pull request merged**